neuromation/cli/root.py

```diff
@@ -19,6 +20,11 @@
 
 TEXT_TYPE = ("application/json", "text")
 
+HEADER_TOKEN_PATTERNS = [
+    re.compile(rf"({auth_scheme})\s+([^ ]+\.[^ ]+\.[^ ]+)")
```

Comment:
We also can simplify the patterns not to know about JWT token format: `re.compile(rf"({auth_scheme})\s+([^ ]+")`

Reply:
not the case anymore.